datafusion/execution/src/memory_pool/mod.rs

```diff
@@ -140,9 +143,9 @@ pub trait MemoryPool: Send + Sync + std::fmt::Debug {
 /// [`MemoryReservation`] in a [`MemoryPool`]. All allocations are registered to
 /// a particular `MemoryConsumer`;
 ///
-/// For help with allocation accounting, see the [proxy] module.
+/// For help with allocation accounting, see the [`proxy`] module.
 ///
-/// [proxy]: crate::memory_pool::proxy
+/// [proxy]: datafusion_common::utils::proxy
```

Comment:
`proxy` was moved to datafusion commen in https://github.com/apache/datafusion/pull/10561 / https://github.com/apache/datafusion/commit/b36f5e20207270378f753c2d8c5fc73d8bc80781